services/harmony/test/util/edl-api.ts

```diff
@@ -0,0 +1,34 @@
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import {
+  getEdlGroupInformation,
+} from '../../app/util/edl-api';
+import logger from '../../app/util/log';
+import { stubEdlRequest, token, unstubEdlRequest } from '../helpers/auth';
+
+describe('util/edl-api', function () {
+  describe('getEdlGroupInformation', function () {
+    before(function () {
+      stubEdlRequest(
+        '/oauth/token',
+        { grant_type: 'client_credentials' },
+        token({ accessToken: 'fake_access' }),
+      );
+    });
+    after(function () {
+      unstubEdlRequest();
+    });
+    describe('when the user is not part of the service deployers group', function () {
+      it('returns isServiceDeployer:false', async function () {
+        const groups = await getEdlGroupInformation('joe', logger);
```

Comment:
I think it would be good to use more descriptive user names here. Instead of 'joe' and 'eve', maybe 'non-service-deployer' and 'service-deployer', etc.

Reply:
Probably out of scope for this ticket (there are a lot of these tests that rely on users in groups). We should start coming up with the usernames roy-the-deployer, sue-the-viewer, bradman-the-admin.